packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Basic.Example.tsx

```diff
@@ -68,7 +68,7 @@ export class ContextualMenuBasicExample extends React.Component<any, any> {
                 },
                 {
                   key: 'divider_1',
```

Comment:
Personally, I'd rather we require the key. I don't want us to end up in a situation where we are using the index for a key inside the component.